src/bridges/metalayer/MetalayerBridge.sol

```diff
@@ -65,19 +65,26 @@ contract MetalayerBridge is Ownable, IMetalayerBridge {
         uint32 destinationDomain_ = _getMetalayerDomain(destinationChainId_);
         ReadOperation[] memory emptyReads_ = new ReadOperation[](0);
 
+        currentRefundAddress = refundAddress_;
+
         // NOTE: The transaction reverts if msg.value isn't enough to cover the fee.
-        //       If msg.value is greater than the required fee, the excess is sent to the refund address.
+        //       If msg.value is greater than the required fee, the router refunds excess to this contract,
+        //       which is then forwarded to refundAddress_ via the receive() function.
         IMetalayerRouter(router).dispatch{ value: msg.value }(
-            destinationDomain_, 
-            peer_, 
-            emptyReads_, 
-            payload_, 
-            FinalityState.INSTANT, 
+            destinationDomain_,
+            peer_,
+            emptyReads_,
+            payload_,
+            FinalityState.INSTANT,
```

Comment:
do we want to push this to be a config global to the contract at construction time or settle on this?